python/protobuf.c

```diff
@@ -104,6 +149,52 @@ PyObject *PyUpb_ObjCache_Get(const void *key) {
   }
 }
 
+bool PyUpb_WeakMap_Next(PyUpb_WeakMap *map, const void **key, PyObject **obj,
+                        intptr_t *iter) {
+  uintptr_t u_key;
+  upb_value val;
+  if (upb_inttable_next2(&map->table, &u_key, &val, iter)) {
```

Comment:
I would invert this conditional and denest the happy path

Reply:
Done.